apps/client/src/components/RegisterLookupSummary.js

```diff
@@ -82,14 +60,14 @@ const RegisterLookupSummary = ({
       </Paragraph>
       <Paragraph
         data-testid={LOCATION_RESTRICTION_CITYSCAPE}
-        gutterBottom={displayZoningPlans ? 16 : 0}
+        gutterBottom={!hasSTTR ? 16 : 0}
```

Comment:
You can flip the values after ternary operator and get rid of the `!`.

Reply:
Nice